webpack/components/extensions/HostDetails/Tabs/RepositorySetsTab/RepositorySetsTab.js

```diff
@@ -349,13 +371,16 @@ const RepositorySetsTab = () => {
           <Thead>
             <Tr>
               <Th key="select-all" />
-              {columnHeaders.map(col =>
-                <Th key={col}>{col}</Th>)}
+              <SortableColumnHeaders
+                columnHeaders={columnHeaders}
+                pfSortParams={pfSortParams}
+                columnsToSortParams={COLUMNS_TO_SORT_PARAMS}
+              />
               <Th />
               <Th key="action-menu" />
```

Comment:
Ditto on permissions question above.

Reply:
If the `<Th>` is outside of the SortableColumnHeaders then hiding it isn't a problem; we could just conditionally render it. Another option is just to allow selection but disable the button.  but I think it's fine for now?